tests/phpunit/elementor/app/import-export/test-module.php

```diff
@@ -269,4 +269,26 @@ public static function assert_valid_import_session( $instance, $session_id )
 
 		$instance->assertEquals( $session_id, $import_sessions[0]['session_id'] );
 	}
+
+	public function test_is_native_class__returns_true_on_native_class() {
+		// Arrange
+		$import_export_module = new Module();
+
+		// Act
+		$is_native_class = $import_export_module->is_native_class( 'Elementor\Test' );
+
+		// Assert
+		$this->assertTrue( $is_native_class );
+	}
```

Comment:
I would try to test a real class instead of passing a string manually
maybe test the actual "handle_ajax" method instead...